esrally/telemetry.py

```diff
@@ -568,6 +568,134 @@ def record(self):
                 self.metrics_store.put_doc(doc, level=MetaInfoScope.cluster, meta_data=shard_metadata)
 
 
+class ShardStats(TelemetryDevice):
+    """
+    Gathers different shard per index stats every minute.
```

Comment:
The documentation here refers to the (current) default value of `shard-stats-sample-interval`. Maybe remove this information?